shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -141,7 +143,7 @@ public PlatformPlugin(
     this.platformChannel.setPlatformMessageHandler(mPlatformMessageHandler);
     this.platformPluginDelegate = delegate;
 
-    mEnabledOverlays = DEFAULT_SYSTEM_UI;
+    currentSystemUiMode = PlatformChannel.SystemUiMode.EDGE_TO_EDGE;
```

Comment:
Is this a new default?

Reply:
The only difference between this mode and the `DEFAULT_SYSTEM_UI` is the [SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION](https://developer.android.com/reference/android/view/View#SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION) flag, which has the same behavior in the new API as another flag they share, [SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN](https://developer.android.com/reference/android/view/View#SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN), so I set this as the default as it appears to be the same.